packages/cloud_functions/cloud_functions/ios/cloud_functions.podspec

```diff
@@ -25,10 +25,6 @@ A new flutter plugin project.
   s.dependency 'Firebase/Functions', '~> 6.0'
   s.static_framework = true
 
-  s.prepare_command = <<-CMD
-      echo // Generated file, do not edit > Classes/UserAgent.h
-      echo "#define LIBRARY_VERSION @\\"#{libraryVersion}\\"" >> Classes/UserAgent.h
-      echo "#define LIBRARY_NAME @\\"flutter-fire-fn\\"" >> Classes/UserAgent.h
-    CMD
+  s.pod_target_xcconfig = { 'GCC_PREPROCESSOR_DEFINITIONS' => '$(inherited) LIBRARY_VERSION=@\"unknown\" LIBRARY_NAME=@\"flutter-fire-fn\"' }
```

Comment:
Is it possible to use #{libraryVersion} here instead of "unknown"?

Reply:
Done. It wasn't converting the `#{libraryVersion} ` to the actual value so I concatenated the string.